src/main/java/com/teamscale/buildbreaker/teamscale/client/exceptions/CommitResolutionExceptionBase.java

```diff
@@ -0,0 +1,13 @@
+package com.teamscale.buildbreaker.teamscale.client.exceptions;
+
+public abstract class CommitResolutionExceptionBase extends Exception {
```

Comment:
New base class for CommitCouldNotBeResolvedException and TooManyCommitsException